documents/src/pages/elements/datetime-picker.md

```diff
@@ -359,23 +358,27 @@ document.getElementById('today').addEventListener('tap', () => {
   const to = new Date().setSeconds(0, 0);
   const from = new Date(to).setHours(0, 0, 0, 0);
   rangePicker.values = toValues(from, to);
+  rangePicker.views = toValues(from, to);
```

Comment:
Let's format view correctly following the warning notice:
> "Warning notice:
The specified value "2024-12-13T11:49" does not conform to the required format. The format is "yyyy-MM".

Reply:
Updated krub.